cmake/modules/Finduring.cmake

```
@@ -0,0 +1,21 @@
+# - Find uring
+#
+# URING_INCLUDE_DIR - Where to find liburing.h
+# URING_LIBRARIES - List of libraries when using uring.
+# uring_FOUND - True if uring found.
+
+find_path(URING_INCLUDE_DIR liburing.h)
+find_library(URING_LIBRARIES liburing.a)
```

```suggestion
find_library(URING_LIBRARIES liburing.a liburing)
```

otherwise, it will not able to work with at least fc33. see https://fedora.pkgs.org/rawhide/fedora-x86_64/liburing-0.5-1.fc33.i686.rpm.html. i am curious what distro you test ?

---

@tchaikov 
Thanks. I'll use your method to solve it.
I use master branch with head commit d3d1ea55

---

i mean what GNU/Linux distribution you test this change with?

---

@tchaikov 
```
ceph$ cat /etc/os-release 
NAME="Ubuntu"
VERSION="18.04.2 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.2 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
ceph$ dpkg -l | grep liburing
ii  liburing-dev:amd64                            0.4-1                                        amd64        userspace library for using io_uring
ii  liburing1:amd64                               0.4-1                                        amd64        userspace library for using io_uring
```
The kernel version should be 5.3.8 (my server is power down, I can't check it right now)

---

i see. you are very likely using liburing-dev compiled from https://github.com/axboe/liburing/tree/master/debian . the reason i asked was that, because AFAIK, fedora does not encourage packaging static libraries. see https://docs.fedoraproject.org/en-US/packaging-guidelines/#_packaging_static_libraries

debian policy allows this. see https://www.debian.org/doc/debian-policy/ch-sharedlibs.html#static-libraries, but it's an addendum to the shared library in general.

---

@tchaikov 
Do I need to do other changes to refine this PR?
The latest PR has been changed to be:
```
find_library(URING_LIBRARIES liburing.a liburing)
```